services/twitch/twitch-extension-version.service.js

```diff
@@ -0,0 +1,57 @@
+import Joi from 'joi'
+import TwitchBase from './twitch-base.js'
+
+const helixSchema = Joi.object({
+  data: Joi.array().required(),
+})
+
+export default class TwitchExtensionVersion extends TwitchBase {
+  static category = 'version'
+
+  static route = {
+    base: 'twitch/extension/v',
+    pattern: ':extensionId',
+  }
+
+  static examples = [
+    {
+      title: 'Twitch Extension Version',
+      namedParams: {
+        extensionId: '2nq5cu1nc9f4p75b791w8d3yo9d195',
+      },
+      staticPreview: this.render({ version: '1.0.0' }),
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'twitch extension',
+  }
+
+  static render({ version }) {
+    return {
+      label: 'twitch extension',
+      message: `v${version}`,
+      color: '6441A4',
+    }
+  }
```

Comment:
The default colour scheme should convey meaning rather than branding.
Can we use `renderVersionBadge()` here (you can search for other examples) instead of a custom `render()`.
If users want to override the default with a brand colour, they can